src/pages/workspace/distanceRates/PolicyDistanceRatesSettingsPage.tsx

```diff
@@ -65,10 +65,7 @@ function PolicyDistanceRatesSettingsPage({route}: PolicyDistanceRatesSettingsPag
             return;
         }
 
-        Category.setPolicyDistanceRatesDefaultCategory(policyID, customUnit, {
-            ...customUnit,
-            defaultCategory: defaultCategory === category.searchText ? '' : category.searchText,
-        });
+        Category.setPolicyCustomUnitDefaultCategory(policyID, customUnitID, customUnit.defaultCategory, defaultCategory === category.searchText ? '' : category.searchText);
```

Comment:
Do we remove the default category if we select the already selected category?

Backend rejects passing an empty category here.
<img width="827" alt="Screenshot 2024-11-13 at 9 03 26 AM" src="https://github.com/user-attachments/assets/e7ee20c0-21d7-48ab-9462-b91e593b8080">

This works fine on staging.

Reply:
There is no UI flow where we pass an empty category, right?

Reply:
There is!


https://github.com/user-attachments/assets/590afee9-6c23-4179-99bb-f995386d6122

Reply:
Oh, interesting. 
I think the issue is frontend here, right? We shouldn't be calling the command if we are not updating anything

Reply:
Clicking on it could mean that we can remove the default category, i.e., reset the category to blank. Is this allowed?

Reply:
Not really. Even when the Custom Unit is newly created, in the UI, it kinda defaults top the first category (You can see the flow in OldDot)